src/spid_const.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
A lot of keys in this file doesn't follow any naming convention, leading to a lot of linter errors. To prevent changing code functionality i disabled the rule explicitly.